data/gloperate/qml/PipelineView.qml

```diff
@@ -39,6 +41,33 @@ Item
                 page.closed();
             }
         }
+
+        Rectangle
+        {
+            x:      100
+            y:      100
+            width:  200
+            height: 200
+
+            color:        'white'
+            border.color: 'black'
+            border.width: 1
+
+            TextureItem
+            {
+                anchors.fill: parent
+                anchors.margins: 1
+
+                //path: 'DemoPipeline.TextureLoadStage.texture'
+                path: 'DemoPipeline.FramebufferStage1.colorTexture'
```

Comment:
I assume this will be a property once the architecture on how to register / propagate render targets is clear?